storage/block/block.go

```diff
@@ -294,8 +290,6 @@ func (b *dbBlock) ResetRetrievable(
 }
 
 func (b *dbBlock) stream(ctx context.Context) (xio.BlockReader, error) {
-	b.ctx.DependsOn(ctx)
```

Comment:
Did we have a double DependsOn? I see one in Stream() too...that might be my bad

Reply:
I saw that, yeah wasn't 100% sure but it may have been a double.  Either way we can get rid of that complexity with this change thankfully.